charts/k8s-ttl-controller/values.yaml

```diff
@@ -3,6 +3,14 @@ image:
   tag: v1.2.0
   pullPolicy: IfNotPresent
 
+resources:
+  requests:
+    cpu: 16m
+    memory: 128Mi
+  limits:
+    cpu: 32m
+    memory: 256Mi
```

Comment:
I would prefer for the default to be no resources specified, so could you comment this by default? 🙏